Added collapsable components under Channels, Direct Messages, and Private Groups

bbrockman opened the pull request:
Added collapsable components under Channels, Direct Messages, and Private Groups
Added Kadira Debug
Added Boostrap - Included only required components for collapsible functionality
Added styles for collapsable components

bbrockman committed:
Added Kadira Debug
Added Boostrap - Included only required components for collapsible functionality
Added styles for collapsable components
Added collapsable components under Channels, Direct Messages, and Private Groups

rwakida commented:
Verified it expands/collapses.  :+1: 

Suggestions:
1. Can we show the arrow only if there are items to collapse.  e.g. if user is not a channel member, then show "You aren't part of any channel yet" message and hide arrow.  
2. either move the arrow to be right-aligned with section's '+' or nudge to the right so that it's left-aligned with section title.

bbrockman commented:
Good ideas.  

Just finished your first suggestion.  I'll take a look at your second.

bbrockman committed:
Updated to reflect Reid's suggestions on the pull request: sscpac/chat-locker/pull/40.
1) Added logic to collapse to make sure there was at least 1 room/direct message/channel before adding the collapsible icon.
2) Aligned the collapsable icons to the icons above them.

bbrockman commented:
Updated them both.  

Reid,
  Can you check out the updates to see if you agree with the changes.

rwakida commented:
nice. :+1:

douglasrapp commented:
I like it. Looks good to me.

douglasrapp commented on client/lib/custom.bootstrap.mixins.import.less:
```diff
@@ -0,0 +1,1151 @@
+    right: percentage((@columns / @grid-columns));
+  }
+}
+
```
I like how 99% of the code change in this pull request is adding giant bootstrap files ;)

bbrockman replied:
All so I can use the collapse functionality.

javawebdeveloper committed:
Merge branch 'master' into feature/chat-locker-32

# Conflicts:
#	.meteor/packages
#	client/stylesheets/base.less

bbrockman committed:
Merge remote-tracking branch 'origin/master' into feature/chat-locker-32

* origin/master:
  Fixed Indenting error!  Thanks Coffee!
  Added comment to not add additional fields to be returned with the publication because it now returns all users.
  Removed logging
  Updated onlineUsers to allUsers for rooms because we want to see everyone in the room Removed offset Added profile first name and last name to user data publication Updated userObserve to include new profile fields Updated room to display first name and last name (sort by last name) instead of username
  Added status-{{status}} to paragraph that displayed username in Member List. In room.coffee: returned offline for the status in the method userActiveByUsername.  The method previously pulled the user from the session when doing the check.  It disregarded it if the user was not in the session (offline). Commented out the base.less file that was overriding the status color in the member list.

bbrockman committed:
Updated to latest versions.

bbrockman committed:
Fixed missing semicolon
